File: web-dashboard/app/Models/Employee.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Employee extends Model
{
    protected $table = 'employees';

    protected $fillable = [
        'name',
        'number_employee',
        'phone',
        'departement_id',
    ];

    public function departement()
    {
        return $this->belongsTo(Departement::class);
    }
}

File: android-app/app/src/main/java/com/example/visitormanagementapp/model/Employee.kt
package com.example.visitormanagementapp.model

import com.google.gson.annotations.SerializedName

data class Employee(
    val id: Int,
    val name: String?,
    @SerializedName("number_employee")
    val numberEmployee: String?,
    @SerializedName("departement_id")
    val departemenId: Int?,
    @SerializedName("phone")
    val phoneNumber: String?
)
